lib/index.js

```diff
@@ -77,7 +77,9 @@ function GracefulShutdown(server, opts) {
     debug('received shut down signal', signal);
     shutdown(signal)
       .then(() => {
-        process.exit(failed ? 1 : 0);
+        if (opts.forceExit) {
```

Comment:
is this a bug in the underlying library, 'cause the option isn't used there either: https://github.com/sebhildebrandt/http-graceful-shutdown/blob/v3.1.7/lib/index.js

Reply:
It is. but I think I introduced it ... should I just pull request against that? I did a sync from the parent repo today as well